packages/config-loader/src/lib/schema/types.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AppConfig, JsonObject } from '@backstage/config';
+
+/**
+ * An sub-set of configuration schema.
+ */
+export type ConfigSchemaPackageEntry = {
+  /**
+   * The configuration schema itself, as JSONSchema draft-07
```

Comment:
Wait but, this is not the meta-schema right? Either way, it's technically the superset of draft-07, could link to the actual schema here instead

Reply:
Type would actually be embedded in the json data rather than be defined here anyway. Got rid of the rest of the comment, was from when It was actually using a proper schema type from ajv.